docs/examples/custom/README.md

```diff
@@ -311,6 +322,8 @@ Now that we've built a custom image and verified that it works as expected, we c
 There is a large number of tools out there to deploy images.
 However, for our example, we will focus on deploying it to a cluster running [Seldon Core](https://docs.seldon.io/projects/seldon-core/en/latest/).
 
+Also consider that depending on your Kubernetes installation Seldon Core might excpect to get the container image from a public container registry like [Docker hub](https://hub.docker.com/) or [Google Container Registry](https://cloud.google.com/container-registry). For that you need to do an extra step of pushing the container to the registry using `docker tag <image name> <container registry>/<image name>` and `docker push <container registry>/<image name>` and also updating the `image` section of the yaml file to `<container registry>/<image name>`. 
```

Comment:
Would it make sense to add this as a `note` block instead? It feels like, that way, it may be less distracting from the main the main walkthrough of what's going on (i.e. as something else to just keep in mind). 

This can be done adding a "code block" in markdown, tagged as `{note}`, as in:

````
```{note}
bli bla blu
```
````

Reply:
Hey @saeid93 ,

Not sure I follow.

It's possible to have multi-line code blocks within notes. Just to clarify, under the hood we use `myst`, which is a markdown preprocessor to render the docs with Sphinx. So it will turn the `note` block to a visual box, separate from the main text.

My main concern is that, otherwise, the current clarification breaks the previous flow between its adjacent paragraphs. You can see that the next paragraph starts with `For that, we will need to create a...`, which, after the change, seems to refer to the action of pushing the image instead of deploying the model.

Since the mention to pushing the image is mainly a clarification, I think it makes sense to move it to a separate note. As it's mainly something to keep in mind and not a specific call to action.

Reply:
Sure, I almost forgot that it will be also used for generating the webpages and not just for the jupyter notebook. Please check the new one.